devtools/conda-envs/asapdiscovery-macOS-latest.yml

```diff
@@ -87,7 +87,7 @@ dependencies:
     - validators
     - pyyaml
     # ml
-    - git+https://github.com/choderalab/mtenn@0.4.0
+    - git+https://github.com/choderalab/mtenn@unify-conversion-utils
```

Comment:
Do we need to move this pin back prior to merge as noted in the start of this PR.

Reply:
my plan is to first merge choderalab/mtenn#34, cut a 0.5.0 release of `mtenn`, and then update this pin